src/archivematicaCommon/lib/databaseFunctions.py

```diff
@@ -356,3 +358,17 @@ def deUnicode(str):
     if str is None:
         return None
     return unicode(str).encode('utf-8')
+
+
+def create_directory_models(dir_paths_uuids, unit_mdl, unit_type='transfer'):
```

Comment:
I often call "model class instances" just "models" as a shorthand; given the context, I think it makes sense. Similarly, I informally type-annotate such instances with `_mdl`, e.g., `file_mdl = File()` if I think it improves readability. But I guess you're right that in this case it kinda sounds like I'm dynamically create Django classes, so I'll change it.